Formula/slowhttptest.rb

```diff
@@ -13,15 +14,23 @@ class Slowhttptest < Formula
     sha256 "c9b36ccf8aee0f6572e2eb1112caf25811c22f1500ad81c1277309c76bd6460b" => :high_sierra
   end
 
+  # Remove these in next version
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+
   depends_on "openssl@1.1"
 
   def install
+    system "sed", "-i", "''", "s/1\.8\.1/1\.8\.2/", "configure.ac" # Remove in next version
```

Comment:
can you use `inreplace` instead. it'll fail if it can't find it.

Reply:
Done